sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3724,7 +3725,10 @@ SUBROUTINE MISCLN
                   HELI(ista_2l:iend_2u,jsta_2l:jend_2u,2))
          allocate(LLOW(ista_2l:iend_2u,jsta_2l:jend_2u),LUPP(ista_2l:iend_2u,jsta_2l:jend_2u),   &
                   CANGLE(ista_2l:iend_2u,jsta_2l:jend_2u))
-
+         allocate(LLOW_ZINT(ista_2l:iend_2u,jsta_2l:jend_2u), 					 &
+                  IEQL_ZINT(ista_2l:iend_2u,jsta_2l:jend_2u),Z_TEMP(ista_2l:iend_2u,jsta_2l:jend_2u))
+         allocate(MIDCAL(ista_2l:iend_2u,jsta_2l:jend_2u,1:LM))
+         allocate(Z_MIDCAL(ista_2l:iend_2u,jsta_2l:jend_2u))
```

Comment:
@EdwardColon-NOAA Can you add changes for deallocating these variables at the end of this subroutine?

Reply:
@EdwardColon-NOAA Could you address my suggestion above?